.github/workflows/check_wdk.yml

```
@@ -0,0 +1,62 @@
+  # Run script every Sunday at midnight
+  schedule:
+    - cron: '0 0 * * 0'
+  # Allow manual triggering of the script
+  workflow_dispatch:
+
+jobs:
+  check:
+    runs-on: Windows-latest
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      if: steps.skip_check.outputs.should_skip != 'true'
```

actionlint (1.7.4)

property "skip_check" is not defined in object type {}

(expression)